apps/common/send_email.py

```diff
@@ -0,0 +1,40 @@
+from fastapi_mail import FastMail, MessageSchema, ConnectionConfig, MessageType
```

Comment:
I have error with fastapi_mail, so i desided to do `poerty update`, after that i see this error
```
Because fastapi-mail (1.2.2) depends on starlette (>=0.21.0,<0.22.0)
 and fastapi (0.78.0) depends on starlette (0.19.1), fastapi-mail (1.2.2) is incompatible with fastapi (0.78.0).
So, because e-university-api depends on both fastapi (0.78.0) and fastapi-mail (1.2.2), version solving failed.

```

Reply:
fixed